METRICS_CONFIGURATION.md

```
@@ -0,0 +1,313 @@
+
+Metrics are **disabled by default**. To enable metrics collection, set `METRICS_ENABLED=true` and start GOModel:
+
+```bash
+export METRICS_ENABLED=true
+./bin/gomodel
+# Metrics available at http://localhost:8080/metrics
+```
+
+### Disable Metrics
+
+**Option 1: Environment Variable**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)